src/plugins/intel_cpu/tests/functional/shared_tests_instances/single_layer_tests/scatter_ND_update.cpp

```diff
@@ -62,4 +63,54 @@ const auto ScatterNDUpdateCases = ::testing::Combine(
 
 INSTANTIATE_TEST_SUITE_P(smoke_ScatterNDUpdate, ScatterNDUpdateLayerTest, ScatterNDUpdateCases, ScatterNDUpdateLayerTest::getTestCaseName);
 
+const std::vector<ov::op::v15::ScatterNDUpdate::Reduction> reduceModes{
+    ov::op::v15::ScatterNDUpdate::Reduction::SUM,
+    ov::op::v15::ScatterNDUpdate::Reduction::SUB,
+    ov::op::v15::ScatterNDUpdate::Reduction::NONE,
+    ov::op::v15::ScatterNDUpdate::Reduction::PROD,
+    ov::op::v15::ScatterNDUpdate::Reduction::MIN,
+    ov::op::v15::ScatterNDUpdate::Reduction::MAX
+};
+
+// map<input_shape map<indices_shape, indices_value>>
+// update_shape is gotten from input_shape and indices_shape
+std::map<std::vector<size_t>, std::map<std::vector<size_t>, std::vector<int>>> sliceSelectInShape15 = {
+    {{10, 9, 9, 11}, {{{4, 1}, {1, 3, 5, 7}}, {{1, 2}, {4, 6}}, {{2, 3}, {0, 1, 1, 2, 2, 2}}, {{1, 4}, {5, 5, 4, 9}}}},
+    {{10, 9, 10, 9, 10}, {{{2, 2, 1}, {5, 6, 2, 8}}, {{2, 3}, {0, 4, 6, 5, 7, -1}}}},
+};
+const auto ScatterNDUpdate15Cases = ::testing::Combine(
+        ::testing::ValuesIn(combineShapes(sliceSelectInShape15)),
+        ::testing::ValuesIn(reduceModes),
+        ::testing::ValuesIn(model_types),
+        ::testing::ValuesIn(idx_types),
+        ::testing::Values(ov::test::utils::DEVICE_CPU)
+);
+INSTANTIATE_TEST_SUITE_P(smoke_ScatterNDUpdate15, ScatterNDUpdate15LayerTest, ScatterNDUpdate15Cases, ScatterNDUpdate15LayerTest::getTestCaseName);
+
+// Due to undefined order of updates, exclude Reduction::NONE from duplicated indices tests
+const std::vector<ov::op::v15::ScatterNDUpdate::Reduction> reduceModesDuplicate{
+    ov::op::v15::ScatterNDUpdate::Reduction::SUM,
+    ov::op::v15::ScatterNDUpdate::Reduction::SUB,
+    ov::op::v15::ScatterNDUpdate::Reduction::PROD,
+    ov::op::v15::ScatterNDUpdate::Reduction::MIN,
+    ov::op::v15::ScatterNDUpdate::Reduction::MAX
+};
+
+// map<input_shape map<indices_shape, indices_value>>
+// update_shape is gotten from input_shape and indices_shape
+std::map<std::vector<size_t>, std::map<std::vector<size_t>, std::vector<int>>> sliceSelectInShape15Duplicate = {
+    {{1}, {{{1}, {0}}, {{2, 1}, {0, 0}}, {{8, 1}, {0, -1, 0, -1, 0, -1, 0, -1}}}},
```

Comment:
Do I correctly understand that case with duplicated indices (real reduction scenario) is verified only for single ({1}) input shape? Shouldn't we extend the coverage to cover different ranks and sizes?

Reply:
Added couple more tests for duplicated indices to have similar coverage as tests for regular shapes. As for testing reduction on single input, it was useful to detect missing updates when experimenting with parallel implementations